File: wrappers/csharp/Intel.RealSense/Types/CameraInfo.cs
using System;
using System.Collections.Generic;
using System.Runtime.InteropServices;

namespace Intel.RealSense
{
    public enum CameraInfo
    {
        Name = 0,
        SerialNumber = 1,
        FirmwareVersion = 2,
        RecommendedFirmwareVersion = 3,
        PhysicalPort = 4,
        DebugOpCode = 5,
        AdvancedMode = 6,
        ProductId = 7,
        CameraLocked = 8,
        UsbTypeDescriptor = 9,
    }
}

File: wrappers/android/librealsense/src/main/java/com/intel/realsense/librealsense/CameraInfo.java
package com.intel.realsense.librealsense;

public enum CameraInfo {
    NAME(0),
    SERIAL_NUMBER(1),
    FIRMWARE_VERSION(2),
    RECOMMENDED_FIRMWARE_VERSION(3),
    PHYSICAL_PORT(4),
    DEBUG_OP_CODE(5),
    ADVANCED_MODE(6),
    PRODUCT_ID(7),
    CAMERA_LOCKED(8),
    USB_TYPE_DESCRIPTOR(9);

    private final int mValue;

    private CameraInfo(int value) { mValue = value; }
    public int value() { return mValue; }
}